### PR TITLE
VEX-7303: Android: Tune memory consumption to prevent libc.so crashes

### PR DESCRIPTION
**Pull request by armadilio3**
Add support for more memory tuning capabilities.

### CODE REVIEW

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java [hunk 1]
```diff
@@ -36,6 +36,7 @@ public class ReactExoplayerViewManager extends ViewGroupManager<ReactExoplayerVi
     private static final String PROP_DRM_TYPE = "type";
     private static final String PROP_DRM_LICENSESERVER = "licenseServer";
     private static final String PROP_DRM_HEADERS = "headers";
+    private static final String PROP_LIMIT_MAX_RESOLUTION = "limitMaxResolution";
```
Comment (gabriel-rivero) [inline android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java:39]: Please add this new props to README.md

**Review by gabriel-rivero — CHANGES_REQUESTED** [review]
working as intended, documentation of the new props is needed

**Comment by armadilio3** [comment]
@gabriel-rivero I added the prop to readme 👍

**Review by gabriel-rivero — APPROVED** [review]
major, looks good

**Pull request merged**